docs/docs-content/clusters/cluster-management/cluster-updates.md

```diff
@@ -140,3 +142,109 @@ You can follow these steps to validate all cluster update approaches.
 
 4. Confirm that the cluster is now using an updated profile.
+
+## Palette Reconciliation Behavior
+
+When cluster profile updates are applied, Palette will apply the changes to the cluster but will not delete any
+resources that are no longer in use. This may be appropriate in some instances, but removing those resources will be
+necessary in others. This can be achieved by utilizing one of the two following methods.
+
+The first would be to utilize a clean deployment with a new cluster profile. The previous version could be deleted once
+the new cluster and cluster profile are deployed. The challenge, however, is when particular resources are in use.
+
+The second option is to use `kubectl` to scale the resource count to 0. While deleting the resource using something like
+K9s is possible, Palette will recreate it immediately.
+
+### Limitations
+
+:::warning
+
+Once you upgrade your cluster to a new Kubernetes version, you will not be able to downgrade. We recommend that, before
+upgrading, you review the information provided in the
+[Kubernetes Upgrades](../../integrations/kubernetes-support.md#kubernetes-upgrades) section.
+
+:::
+
+- You cannot update a cluster while its status is still **Provisioning**.
+
+- You cannot change the Container Network Interface (CNI) of an Edge cluster. You must delete and re-create the cluster
+  instead. For more information about creating an Edge cluster, refer to
+  [Create Cluster Definition](../edge/site-deployment/cluster-deployment.md).
+
+- Avoid skipping minor versions when upgrading the Kubernetes version of a cluster. Refer to the documentation of your
+  Kubernetes distribution for upgrade guidance and follow the recommended upgrade paths.
+
+  - For PXK and PXK-E, refer to
+    [Upgrade Kubeadm Clusters](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/).
+  - For K3s, refer to [K3s Upgrades](https://docs.k3s.io/upgrades#version-specific-caveats)
+  - For RKE2, refer to [RKE2 Manual Upgrades](https://docs.rke2.io/upgrades/manual_upgrade)
+
+### Prerequisites
+
+- An active Kubernetes cluster in Palette.
+- A terminal client
+
+### Enablement
+
+The steps below show how we can use the second option while deploying the Hello Universe application pack and updating
+the cluster profile. In this example, we create an AWS cluster called cluster-update-deletion. We will use Palette and a
+terminal application.
+
+1. Log into Palette and create a cluster profile with two versions. The base layers should be ubuntu-aws, Kubernetes,
+   cni-calico, and csi-aws-ebs. The App Services layer will use the Hello Universe from the Palette Community Registry
+   (OCI).
+
+   The first profile version (1.0.0) should be the Hello Universe 1.2.0 pack with the API disabled.
+
+   The second version (1.1.0) should be the Hello Universe 1.2.0 pack with the API enabled.
+
+2. Build a cluster using the profile version 1.0.0 (takes about 15-20 min)
```

Comment:
```suggestion
1. Log in to Palette and create a cluster profile for an AWS IaaS cluster that includes the Hello Universe pack. For example:

   OS = **ubuntu-aws**
   Kubernetes = **kubernetes**
   Network = **cni-calico**
   Storage = **csi-aws-ebs**
   App services = **hellouniverse** with the API preset option disabled.

2. Build a cluster using this cluster profile version. This will take 15-20 minutes.

```